cars/cars.hpp

```diff
@@ -1,4 +1,4 @@
-#include "zit/zit.hpp"
+#include "../zit/zit.hpp"
```

Comment:
no, that's wrong. All file paths should be root-of-project-relative, not relative to the file.